lib/font-viewer.js

```diff
@@ -73,15 +73,14 @@ export default class FontViewer {
   getAvailableCharacters(callback) {
     return fs.readFile(this.getPath(), function (err, buffer) {
       const chars = []
-      const gindex = {}
-      let face = {}
-      ft.New_Memory_Face(buffer, 0, face)
-      ;({ face } = face)
-      let charcode = ft.Get_First_Char(face, gindex)
+      const face = NewMemoryFace(buffer, 0);
 
-      while (gindex.gindex !== 0) {
+      let charcode = face.getFirstChar()
+      let num = 0
+      while (charcode !== null) {
+        num++
         chars.push(charcode)
-        charcode = ft.Get_Next_Char(face, charcode, gindex)
+        charcode = face.getNextChar(num)
```

Comment:
This currently does not work. Do you have any idea how the API is changed here? 
@Julusian

Reply:
OK. I figured it out for now.